Sources/Private/CoreAnimation/Layers/TextLayer.swift

```diff
@@ -48,6 +64,9 @@ final class TextLayer: BaseCompositionLayer {
     //  - We may be able to support animating `fillColor` by getting clever with layer blend modes
     //    or masks (e.g. use `CoreTextRenderLayer` to draw black glyphs, and then fill them in
     //    using a `CAShapeLayer`).
+
+    // Question 3: Should we somehow initialize LayerContextAnimations at this stage
```

Comment:
I think we can remove the `renderLayer.text = text.text` call from this method -- it only needs to be done once, and `setupAnimations` is fine.

Reply:
Right! Fixed in [01307aa](https://github.com/airbnb/lottie-ios/pull/1723/commits/01307aa7d4bb9ff238810595d05304aab3f31075) ✅